### PR TITLE
Add ARM64 download links to README

### LINKED ISSUE #1065: Need arm64 build
Now that EC2 instances with Graviton2 processors are in GA, we need an arm64 build so customers can run ecs-cli on these instances.

### PR DESCRIPTION
**Pull request by bvtujo**
Generated and verified ARM64 binaries. Links refer to verified URLS in CN and AWS buckets.

Based on @otterley's PR #1066. This adds automatic build and release for arm64 artifacts. Latest (v1.20.0) are built, integ-tested on a separate codebuild project, and pushed to S3 in us-west-2 and cn-north-1.

Resolves #1065.

----
Enter `[N/A]` in the box, if an item is not applicable to your change.

**Testing**
- [x] Unit tests passed
- [x] Integration tests passed
- [n/a] Unit tests added for new functionality
- [x] Listed manual checks and their outputs in the comments ([example](https://github.com/aws/amazon-ecs-cli/pull/750#issuecomment-472623042))
- [n/a] Link to issue or PR for the integration tests:

----

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by bvtujo** [comment]
See #1066.

**Comment by bvtujo** [comment]
GPG signature verification

```
(20-10-01 23:32:55) <0> [~/w/ecs-cli-artifacts/verification/aws]
dev-dsk-austiely-2b-1a7c1482 % wget https://amazon-ecs-cli.s3.amazonaws.com/ecs-cli-linux-arm64-latest
--2020-10-01 23:33:14--  https://amazon-ecs-cli.s3.amazonaws.com/ecs-cli-linux-arm64-latest
Resolving amazon-ecs-cli.s3.amazonaws.com (amazon-ecs-cli.s3.amazonaws.com)... 52.216.93.219
Connecting to amazon-ecs-cli.s3.amazonaws.com (amazon-ecs-cli.s3.amazonaws.com)|52.216.93.219|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 34799616 (33M) [binary/octet-stream]
Saving to: ‘ecs-cli-linux-arm64-latest’

100%[==========================================================>] 34,799,616  14.8MB/s   in 2.2s

2020-10-01 23:33:17 (14.8 MB/s) - ‘ecs-cli-linux-arm64-latest’ saved [34799616/34799616]


(20-10-01 23:33:17) <0> [~/w/ecs-cli-artifacts/verification/aws]
dev-dsk-austiely-2b-1a7c1482 % wget https://amazon-ecs-cli.s3.amazonaws.com/ecs-cli-linux-arm64-latest.asc
--2020-10-01 23:33:21--  https://amazon-ecs-cli.s3.amazonaws.com/ecs-cli-linux-arm64-latest.asc
Resolving amazon-ecs-cli.s3.amazonaws.com (amazon-ecs-cli.s3.amazonaws.com)... 52.216.93.219
Connecting to amazon-ecs-cli.s3.amazonaws.com (amazon-ecs-cli.s3.amazonaws.com)|52.216.93.219|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 821 [text/plain]
Saving to: ‘ecs-cli-linux-arm64-latest.asc’

100%[==========================================================>] 821         --.-K/s   in 0s

2020-10-01 23:33:21 (78.5 MB/s) - ‘ecs-cli-linux-arm64-latest.asc’ saved [821/821]


(20-10-01 23:33:21) <0> [~/w/ecs-cli-artifacts/verification/aws]
dev-dsk-austiely-2b-1a7c1482 % wget https://amazon-ecs-cli.s3.amazonaws.com/ecs-cli-linux-arm64-v1.20.0.asc
--2020-10-01 23:33:27--  https://amazon-ecs-cli.s3.amazonaws.com/ecs-cli-linux-arm64-v1.20.0.asc
Resolving amazon-ecs-cli.s3.amazonaws.com (amazon-ecs-cli.s3.amazonaws.com)... 52.216.93.219
Connecting to amazon-ecs-cli.s3.amazonaws.com (amazon-ecs-cli.s3.amazonaws.com)|52.216.93.219|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 821 [text/plain]
Saving to: ‘ecs-cli-linux-arm64-v1.20.0.asc’

100%[==========================================================>] 821         --.-K/s   in 0s

2020-10-01 23:33:27 (74.8 MB/s) - ‘ecs-cli-linux-arm64-v1.20.0.asc’ saved [821/821]


(20-10-01 23:33:27) <0> [~/w/ecs-cli-artifacts/verification/aws]
dev-dsk-austiely-2b-1a7c1482 % wget https://amazon-ecs-cli.s3.amazonaws.com/ecs-cli-linux-arm64-v1.20.0
--2020-10-01 23:33:32--  https://amazon-ecs-cli.s3.amazonaws.com/ecs-cli-linux-arm64-v1.20.0
Resolving amazon-ecs-cli.s3.amazonaws.com (amazon-ecs-cli.s3.amazonaws.com)... 52.217.65.36
Connecting to amazon-ecs-cli.s3.amazonaws.com (amazon-ecs-cli.s3.amazonaws.com)|52.217.65.36|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 34799616 (33M) [binary/octet-stream]
Saving to: ‘ecs-cli-linux-arm64-v1.20.0’

100%[==========================================================>] 34,799,616  12.5MB/s   in 2.6s

2020-10-01 23:33:35 (12.5 MB/s) - ‘ecs-cli-linux-arm64-v1.20.0’ saved [34799616/34799616]


(20-10-01 23:33:35) <0> [~/w/ecs-cli-artifacts/verification/aws]
dev-dsk-austiely-2b-1a7c1482 % ls
ecs-cli-linux-arm64-latest      ecs-cli-linux-arm64-v1.20.0
ecs-cli-linux-arm64-latest.asc  ecs-cli-linux-arm64-v1.20.0.asc

(20-10-01 23:33:36) <0> [~/w/ecs-cli-artifacts/verification/aws]
dev-dsk-austiely-2b-1a7c1482 % gpg --verify ecs-cli-linux-arm64-latest.asc ecs-cli-linux-arm64-latest
gpg: Signature made Thu 01 Oct 2020 10:58:37 PM UTC using RSA key ID ADAF8B8E
gpg: Good signature from "Amazon ECS <ecs-security@amazon.com>"
gpg: WARNING: This key is not certified with a trusted signature!
gpg:          There is no indication that the signature belongs to the owner.
Primary key fingerprint: F34C 3DDA E729 26B0 79BE  AEC6 BCE9 D9A4 2D51 784F
     Subkey fingerprint: EB3D F841 E2C9 212A 2BD4  2232 DE3C BD61 ADAF 8B8E

(20-10-01 23:34:45) <0> [~/w/ecs-cli-artifacts/verification/aws]
dev-dsk-austiely-2b-1a7c1482 % gpg --verify ecs-cli-linux-arm64-v1.20.0.asc ecs-cli-linux-arm64-v1.20.0
gpg: Signature made Thu 01 Oct 2020 10:58:45 PM UTC using RSA key ID ADAF8B8E
gpg: Good signature from "Amazon ECS <ecs-security@amazon.com>"
gpg: WARNING: This key is not certified with a trusted signature!
gpg:          There is no indication that the signature belongs to the owner.
Primary key fingerprint: F34C 3DDA E729 26B0 79BE  AEC6 BCE9 D9A4 2D51 784F
     Subkey fingerprint: EB3D F841 E2C9 212A 2BD4  2232 DE3C BD61 ADAF 8B8E

(20-10-01 23:35:10) <0> [~/w/ecs-cli-artifacts/verification/aws]
dev-dsk-austiely-2b-1a7c1482 % cat verification_ecs-cli-linux-arm64-latest

(20-10-01 23:35:17) <0> [~/w/ecs-cli-artifacts/verification/aws]
dev-dsk-austiely-2b-1a7c1482 % ls
ecs-cli-linux-arm64-latest      ecs-cli-linux-arm64-v1.20.0.asc
ecs-cli-linux-arm64-latest.asc  verification_ecs-cli-linux-arm64-latest
ecs-cli-linux-arm64-v1.20.0

(20-10-01 23:35:23) <0> [~/w/ecs-cli-artifacts/verification/aws]
dev-dsk-austiely-2b-1a7c1482 % rm verification_ecs-cli-linux-arm64-latest

(20-10-01 23:41:56) <0> [~/w/ecs-cli-artifacts/verification/aws-cn]
dev-dsk-austiely-2b-1a7c1482 % wget https://amazon-ecs-cli.s3.cn-north-1.amazonaws.com.cn/ecs-cli-linux-arm64-latest
--2020-10-01 23:42:25--  https://amazon-ecs-cli.s3.cn-north-1.amazonaws.com.cn/ecs-cli-linux-arm64-latest
Resolving amazon-ecs-cli.s3.cn-north-1.amazonaws.com.cn (amazon-ecs-cli.s3.cn-north-1.amazonaws.com.cn)... 54.222.49.49
Connecting to amazon-ecs-cli.s3.cn-north-1.amazonaws.com.cn (amazon-ecs-cli.s3.cn-north-1.amazonaws.com.cn)|54.222.49.49|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 34799616 (33M) [binary/octet-stream]
Saving to: ‘ecs-cli-linux-arm64-latest’

100%[==========================================================>] 34,799,616  7.99MB/s   in 5.3s

2020-10-01 23:42:32 (6.26 MB/s) - ‘ecs-cli-linux-arm64-latest’ saved [34799616/34799616]


(20-10-01 23:42:32) <0> [~/w/ecs-cli-artifacts/verification/aws-cn]
dev-dsk-austiely-2b-1a7c1482 % wget https://amazon-ecs-cli.s3.cn-north-1.amazonaws.com.cn/ecs-cli-linux-arm64-latest.asc
--2020-10-01 23:42:35--  https://amazon-ecs-cli.s3.cn-north-1.amazonaws.com.cn/ecs-cli-linux-arm64-latest.asc
Resolving amazon-ecs-cli.s3.cn-north-1.amazonaws.com.cn (amazon-ecs-cli.s3.cn-north-1.amazonaws.com.cn)... 54.222.49.30
Connecting to amazon-ecs-cli.s3.cn-north-1.amazonaws.com.cn (amazon-ecs-cli.s3.cn-north-1.amazonaws.com.cn)|54.222.49.30|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 821 [text/plain]
Saving to: ‘ecs-cli-linux-arm64-latest.asc’

100%[==========================================================>] 821         --.-K/s   in 0s

2020-10-01 23:42:36 (68.2 MB/s) - ‘ecs-cli-linux-arm64-latest.asc’ saved [821/821]


(20-10-01 23:42:36) <0> [~/w/ecs-cli-artifacts/verification/aws-cn]
dev-dsk-austiely-2b-1a7c1482 % wget https://amazon-ecs-cli.s3.cn-north-1.amazonaws.com.cn/ecs-cli-linux-arm64-v1.20.0.asc
--2020-10-01 23:42:42--  https://amazon-ecs-cli.s3.cn-north-1.amazonaws.com.cn/ecs-cli-linux-arm64-v1.20.0.asc
Resolving amazon-ecs-cli.s3.cn-north-1.amazonaws.com.cn (amazon-ecs-cli.s3.cn-north-1.amazonaws.com.cn)... 54.222.49.30
Connecting to amazon-ecs-cli.s3.cn-north-1.amazonaws.com.cn (amazon-ecs-cli.s3.cn-north-1.amazonaws.com.cn)|54.222.49.30|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 821 [text/plain]
Saving to: ‘ecs-cli-linux-arm64-v1.20.0.asc’

100%[==========================================================>] 821         --.-K/s   in 0s

2020-10-01 23:42:43 (73.2 MB/s) - ‘ecs-cli-linux-arm64-v1.20.0.asc’ saved [821/821]


(20-10-01 23:42:43) <0> [~/w/ecs-cli-artifacts/verification/aws-cn]
dev-dsk-austiely-2b-1a7c1482 % wget https://amazon-ecs-cli.s3.cn-north-1.amazonaws.com.cn/ecs-cli-linux-arm64-v1.20.0
--2020-10-01 23:42:47--  https://amazon-ecs-cli.s3.cn-north-1.amazonaws.com.cn/ecs-cli-linux-arm64-v1.20.0
Resolving amazon-ecs-cli.s3.cn-north-1.amazonaws.com.cn (amazon-ecs-cli.s3.cn-north-1.amazonaws.com.cn)... 54.222.49.30
Connecting to amazon-ecs-cli.s3.cn-north-1.amazonaws.com.cn (amazon-ecs-cli.s3.cn-north-1.amazonaws.com.cn)|54.222.49.30|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 34799616 (33M) [binary/octet-stream]
Saving to: ‘ecs-cli-linux-arm64-v1.20.0’

100%[==========================================================>] 34,799,616  7.56MB/s   in 5.6s

2020-10-01 23:42:54 (5.98 MB/s) - ‘ecs-cli-linux-arm64-v1.20.0’ saved [34799616/34799616]


(20-10-01 23:42:54) <0> [~/w/ecs-cli-artifacts/verification/aws-cn]
dev-dsk-austiely-2b-1a7c1482 % gpg --verify ecs-cli-linux-arm64-latest.asc ecs-cli-linux-arm64-latest > verification_ecs-cli-linux-arm64-latest
gpg: Signature made Thu 01 Oct 2020 10:58:37 PM UTC using RSA key ID ADAF8B8E
gpg: Good signature from "Amazon ECS <ecs-security@amazon.com>"
gpg: WARNING: This key is not certified with a trusted signature!
gpg:          There is no indication that the signature belongs to the owner.
Primary key fingerprint: F34C 3DDA E729 26B0 79BE  AEC6 BCE9 D9A4 2D51 784F
     Subkey fingerprint: EB3D F841 E2C9 212A 2BD4  2232 DE3C BD61 ADAF 8B8E

(20-10-01 23:44:35) <0> [~/w/ecs-cli-artifacts/verification/aws-cn]
dev-dsk-austiely-2b-1a7c1482 % gpg --verify ecs-cli-linux-arm64-v1.20.0.asc ecs-cli-linux-arm64-v1.20.0
gpg: Signature made Thu 01 Oct 2020 10:58:45 PM UTC using RSA key ID ADAF8B8E
gpg: Good signature from "Amazon ECS <ecs-security@amazon.com>"
gpg: WARNING: This key is not certified with a trusted signature!
gpg:          There is no indication that the signature belongs to the owner.
Primary key fingerprint: F34C 3DDA E729 26B0 79BE  AEC6 BCE9 D9A4 2D51 784F
     Subkey fingerprint: EB3D F841 E2C9 212A 2BD4  2232 DE3C BD61 ADAF 8B8E

```

File: README.md [hunk 1]
```diff
@@ -94,9 +100,12 @@ same binaries, but they are localized within China to provide a faster download
 Using the URLs above, replace `latest` with the desired tag, for example `v1.0.0`. After
 downloading, remember to rename the binary file to `ecs-cli`. ***NOTE:*** Windows is only supported starting with version `v1.0.0`.
```
Comment (bvtujo) [inline README.md]:
```suggestion
downloading, remember to rename the binary file to `ecs-cli`. 

***NOTE:*** Windows is only supported starting with version `v1.0.0`.

***NOTE:*** ARM64 is only supported starting with version `v1.20.0`.
```

File: README.md [hunk 2]
```diff
@@ -94,9 +100,12 @@ same binaries, but they are localized within China to provide a faster download
...
 
-* Linux:
+* Linux (x86-64):
   * [https://amazon-ecs-cli.s3.amazonaws.com/ecs-cli-linux-amd64-v1.0.0](https://amazon-ecs-cli.s3.amazonaws.com/ecs-cli-linux-amd64-v1.0.0)
   * [https://amazon-ecs-cli.s3.amazonaws.com/ecs-cli-linux-amd64-v1.0.0.md5](https://amazon-ecs-cli.s3.amazonaws.com/ecs-cli-linux-amd64-v1.0.0.md5)
+* Linux (arm64):
+  * [https://amazon-ecs-cli.s3.amazonaws.com/ecs-cli-linux-arm64-v1.0.0](https://amazon-ecs-cli.s3.amazonaws.com/ecs-cli-linux-arm64-v1.0.0)
```
Comment (bvtujo) [inline README.md]:
```suggestion
  * [https://amazon-ecs-cli.s3.amazonaws.com/ecs-cli-linux-arm64-v1.20.0](https://amazon-ecs-cli.s3.amazonaws.com/ecs-cli-linux-arm64-v1.20.0)
```

File: README.md [hunk 3]
```diff
@@ -94,9 +100,12 @@ same binaries, but they are localized within China to provide a faster download
...
+  * [https://amazon-ecs-cli.s3.amazonaws.com/ecs-cli-linux-arm64-v1.0.0.md5](https://amazon-ecs-cli.s3.amazonaws.com/ecs-cli-linux-arm64-v1.0.0.md5)
```
Comment (bvtujo) [inline README.md]:
```suggestion
  * [https://amazon-ecs-cli.s3.amazonaws.com/ecs-cli-linux-arm64-v1.20.0.md5](https://amazon-ecs-cli.s3.amazonaws.com/ecs-cli-linux-arm64-v1.20.0.md5)
```

File: README.md [hunk 4]
```diff
@@ -65,9 +65,12 @@ You can use the provided `md5` hash to verify the integrity of your download.
 For information about installing and using the Amazon ECS CLI, see the [ECS Command Line Interface](http://docs.aws.amazon.com/AmazonECS/latest/developerguide/ECS_CLI.html).
 
 ### Latest version
-* Linux:
+* Linux (x86-64):
   * [https://amazon-ecs-cli.s3.amazonaws.com/ecs-cli-linux-amd64-latest](https://amazon-ecs-cli.s3.amazonaws.com/ecs-cli-linux-amd64-latest)
   * [https://amazon-ecs-cli.s3.amazonaws.com/ecs-cli-linux-amd64-latest.md5](https://amazon-ecs-cli.s3.amazonaws.com/ecs-cli-linux-amd64-latest.md5)
+* Linux (arm64):
+  * [https://amazon-ecs-cli.s3.amazonaws.com/ecs-cli-linux-arm64-latest](https://amazon-ecs-cli.s3.amazonaws.com/ecs-cli-linux-arm64-latest)
+  * [https://amazon-ecs-cli.s3.amazonaws.com/ecs-cli-linux-arm64-latest.md5](https://amazon-ecs-cli.s3.amazonaws.com/ecs-cli-linux-arm64-latest.md5)
```
Comment (piradeepk) [inline README.md]: Should these be identical?

File: README.md [hunk 5]
```diff
@@ -65,9 +65,12 @@ You can use the provided `md5` hash to verify the integrity of your download.
 For information about installing and using the Amazon ECS CLI, see the [ECS Command Line Interface](http://docs.aws.amazon.com/AmazonECS/latest/developerguide/ECS_CLI.html).
 
 ### Latest version
-* Linux:
+* Linux (x86-64):
   * [https://amazon-ecs-cli.s3.amazonaws.com/ecs-cli-linux-amd64-latest](https://amazon-ecs-cli.s3.amazonaws.com/ecs-cli-linux-amd64-latest)
```
Comment (piradeepk) [inline README.md:69]: Shouldn't we build a different binary for x86?

File: README.md [hunk 6]
```diff
@@ -92,11 +98,18 @@ same binaries, but they are localized within China to provide a faster download
 
 ### Download specific version
 Using the URLs above, replace `latest` with the desired tag, for example `v1.0.0`. After
-downloading, remember to rename the binary file to `ecs-cli`. ***NOTE:*** Windows is only supported starting with version `v1.0.0`.
+downloading, remember to rename the binary file to `ecs-cli`. 
 
-* Linux:
+***NOTE:*** Windows is only supported starting with version `v1.0.0`.
+
+***NOTE:*** ARM64 is only supported starting with version `v1.20.0`.
```
Comment (piradeepk) [inline README.md:105]: Is this true?

File: README.md [hunk 7]
```diff
@@ -80,9 +83,12 @@ For information about installing and using the Amazon ECS CLI, see the [ECS Comm
 As of v0.6.2 the ECS CLI supports the cn-north-1 region in China. The following links are the exact
 same binaries, but they are localized within China to provide a faster download experience.
 
-* Linux:
+* Linux (x86-64):
   * [https://amazon-ecs-cli.s3.cn-north-1.amazonaws.com.cn/ecs-cli-linux-amd64-latest](https://amazon-ecs-cli.s3.cn-north-1.amazonaws.com.cn/ecs-cli-linux-amd64-latest)
   * [https://amazon-ecs-cli.s3.cn-north-1.amazonaws.com.cn/ecs-cli-linux-amd64-latest.md5](https://amazon-ecs-cli.s3.cn-north-1.amazonaws.com.cn/ecs-cli-linux-amd64-latest.md5)
+* Linux (arm64):
+  * [https://amazon-ecs-cli.s3.cn-north-1.amazonaws.com.cn/ecs-cli-linux-arm64-latest](https://amazon-ecs-cli.s3.cn-north-1.amazonaws.com.cn/ecs-cli-linux-arm64-latest)
+  * [https://amazon-ecs-cli.s3.cn-north-1.amazonaws.com.cn/ecs-cli-linux-arm64-latest.md5](https://amazon-ecs-cli.s3.cn-north-1.amazonaws.com.cn/ecs-cli-linux-arm64-latest.md5)
```
Comment (piradeepk) [inline README.md]: Same comment as above

**Review by piradeepk — APPROVED** [review]
Ship it! 💯 Looks great!

**Pull request merged**